src/Kmd.Logic.FileSecurity.Client/LogicHttpClientProvider.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Kmd.Logic.FileSecurity.Client.Types;
+using Kmd.Logic.Identity.Authorization;
+using Microsoft.Rest;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+
+namespace Kmd.Logic.FileSecurity.Client
+{
+    public class LogicHttpClientProvider
```

Comment:
Not needed. We can use `LogicTokenProviderFactory` from https://github.com/kmdlogic/kmd-logic-identity-authorization

Reply:
Removed the class.